packages/client/components/Action/Action.tsx

```diff
@@ -33,53 +35,75 @@ const Action = memo(() => {
   useTrebuchetEvents()
   useServiceWorkerUpdater()
   const isInternalAuthEnabled = window.__ACTION__.AUTH_INTERNAL_ENABLED
+  // Global Banner
+  const isGlobalBannerEnabled = window.__ACTION__.GLOBAL_BANNER_ENABLED
+  const bannerText = window.__ACTION__.GLOBAL_BANNER_TEXT
+  const bannerBgColor = window.__ACTION__.GLOBAL_BANNER_BG_COLOR
+  const bannerColor = window.__ACTION__.GLOBAL_BANNER_COLOR
   return (
     <>
       <ErrorBoundary>
         <Snackbar />
         <Suspense fallback={<LoadingComponent spinnerSize={LoaderSize.WHOLE_PAGE} />}>
           <AnalyticsPage />
-          <Switch>
-            <Route exact path='/' render={(p) => <AuthenticationPage {...p} page={'signin'} />} />
-            <Route
-              exact
-              path={`/${SIGNIN_SLUG}`}
-              render={(p) => <AuthenticationPage {...p} page={'signin'} />}
-            />
-            <Route
-              exact
-              path={`/${CREATE_ACCOUNT_SLUG}`}
-              render={(p) => <AuthenticationPage {...p} page={'create-account'} />}
-            />
-            <Route exact path={`/auth/:provider`} component={AuthProvider} />
-            <Route path={`/saml-redirect`} component={SAMLRedirect} />
-            <Route
-              path='/retrospective-demo/:localPhaseSlug?/:stageIdxSlug?'
-              component={DemoMeeting}
-            />
-            <Route path='/retrospective-demo-summary/:urlAction?' component={DemoSummary} />
-            {isInternalAuthEnabled && (
-              <Route
-                exact
-                path={`/forgot-password`}
-                render={(p) => <AuthenticationPage {...p} page={'forgot-password'} />}
-              />
+          <div className='flex h-full max-h-screen flex-col'>
```

Comment:
in testing, i deleted this div & everything still worked OK, is there a use case where this is required?

Reply:
This one allows for better control of the main height by wrapping both the banner and the main view. I think there may be one other way I could try. Let me test.